tests/startup/test_cli.py

```diff
@@ -151,3 +153,60 @@ def test_enterprise_disabled(self):
                 ''')
             # MQTT
             self.assert_mqtt_port(lambda x: x > 0)
+
+    def test_startup_logs(self):
+
+        # Create CRATE_HOME directory
+        tmp_home = self.mkdtemp()
+        Path(tmp_home, 'config').mkdir()
+
+        nodeName = self.fake.last_name()
+        settings = dict({
+            'node.name': nodeName,
+            'path.home': tmp_home,
+        })
+
+        logFilePath = Path(tmp_home, 'crate.log')
+        # Copy log4j2.properties and replace path to crate.log output file
+        with open(Path(Path(__file__).parent, 'log4j2_appendtofile.properties'), 'r') as f:
+            newlines = []
+            for line in f.readlines():
+                newlines.append(line.replace('<log_file_path>', str(logFilePath)))
+        with open(Path(tmp_home, 'config', 'log4j2.properties'), 'w') as f:
+            for line in newlines:
+                f.write(line)
+
+        (node, version_tuple)= self._new_node(self.CRATE_VERSION, settings=settings)
+        node.start()
+
+        # Check entries in log file
+        def verifyAndExtractContentFromLogLine(logLine):
+            self.assertTrue(' [' + nodeName + '] ' in logLine, 'line does not contain correct node name')
+            if '[o.e.n.Node               ]' in line:
+                return logLine.split(' [' + nodeName + '] ')[1]
+            return None
+
+        with open(logFilePath, 'r') as f:
+            count = 1
+            for line in f.readlines():
```

Comment:
could use `for idx, line in enumerate(f):` to avoid the extra count variable and the list creation by `readlines`.

Might also want to put the assertion logic into a separate function? To decouple IO from the pure logic